qframe.go

```diff
@@ -1093,7 +1094,7 @@ func ReadSQL(tx *sql.Tx, confFuncs ...qsql.ConfigFunc) QFrame {
 //
 // This is function is currently unoptimized. It could probably be a lot speedier with
 // a custom written CSV writer that handles quoting etc. differently.
-func (qf QFrame) ToCSV(writer io.Writer) error {
+func (qf QFrame) ToCSV(writer io.Writer, header bool) error {
```

Comment:
For most other functions that provide configurability make use of functional options (https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis). The upside of this is that you can add more options in a backwards compatible manner. The current update breaks compatibility.
See for example https://github.com/tobgu/qframe/blob/master/qframe.go#L383 for guidance. There will be a bit more code to write but it'll add flexibility.

Reply:
Agreed. Need to experiment a bit with it and get back.

Reply:
Agreed

Reply:
Thank you for the article, a good read, although I fear I have not grasped its full intention. 
The example you provided with a struct for Order made sense though, so I modelled that setup. 

Woudl that work for you?